Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayerDebug.h

```diff
@@ -7,9 +7,70 @@
  */
 
 #pragma once
+#include <AzCore/Math/MathStringConversions.h>
 
 namespace Multiplayer
 {
+    //! Categories for Auditing logs used in MultiplayerDebugAuditTrail
+    enum class AuditCategory
+    {
+        Desync,
+        Input,
+        Event
+    };
+
+    //! @class IMultiplayerAuditingDatum
+    //! @brief IMultiplayerAuditingDatum provides an interface for auditing datums of a given auditing event
```

Comment:
nit: "provides an interface for auditing datums of a given auditing event" seems like too many uses of word "auditing"  - it may be accurate, worth a quick review if this is really the responsibilities of this class.